modules/data-entry/src/main/frontend/src/questionnaire/Question.jsx

```diff
@@ -37,7 +38,17 @@ function Question (props) {
         className={classes.questionHeader}
         />
       <CardContent>
-        {children}
+        { !isEdit && displayDefault && prettyAnswers ?
+          <List>
+            { Array.of(prettyAnswers).flat().map( (item) => {
```

Comment:
Move the extraction of the displayedValue here:
```suggestion
            { Array.of(existingAnswer?.[1]["displayedValue"] ).flat().map( (item) => {
```